### ISSUE TITLE
TCPSocket.gethostbyname is deprecated

### ISSUE THREAD

**Issue by exploide** [mozilla/ssh_scan#528]
With ruby 3.0.1p64 I get the following deprecation warning when running ssh_scan:

```
ssh_scan-0.0.43/lib/string_ext.rb:58: warning: TCPSocket.gethostbyname is deprecated; use Addrinfo.getaddrinfo instead.
```

**Comment by claudijd** [comment]
Just fixed this in master, will bump a release shortly.

**Comment by exploide** [comment]
Appears to be fixed in bf57bcf4a28db652208b2e1acff7686fd409a57b and released in 0.0.44. Thank you!
